main.cpp

```diff
@@ -108,7 +108,7 @@ double calculateTime(double initialTime = 0.0, double finalTime = 0.0)
 /*
  2)
  */
-bool fallingOrStopped(bool clicked, int timesOnStage = 0)
+bool boolMoveOnDisplay(bool clicked, int timesOnStage = 0)
```

Comment:
This function name makes no sense.
Function names describe actions. 
Pick a plain-english action, then convert that action into a function name, by eliminating the spaces in the action, and using camel case. 
example:
```
//action: Pick a plain-english action
//code:
void pickAPlainEnglishAction()
```
- [ ] revise